src/components/Widget/RichTextWidget/RichTextWidget.js

```diff
@@ -1,22 +1,23 @@
-import PropTypes from 'prop-types'
-import React, { Component } from 'react'
-
+import React from 'react'
+import GroupAboutVideoEmbed from 'components/GroupAboutVideoEmbed'
 import './RichText.scss'
 
-const { object } = PropTypes
-
-export default class RichTextWidget extends Component {
-  static propTypes = {
-    settings: object
-  }
-
-  render () {
-    const { group, settings = {} } = this.props
-    return (
-      <div styleName='rich-text'>
-        <h2>{settings.title || `Welcome to ${group.name}!`}</h2>
-        <p>{settings.text || `We're happy you're here! Please take a moment to explore this page to see what's alive in our group. Introduce yourself by clicking Create to post a Discussion sharing who you are and what brings you to our group. And don't forget to fill out your profile - so likeminded new friends can connect with you!`}</p>
-      </div>
-    )
-  }
+export default function RichTextWidget ({ group, settings }) {
+  return (
+    <div styleName='rich-text'>
+      <GroupAboutVideoEmbed uri={settings.embeddedVideoURI} />
```

Comment:
so i believe this is where we decided not to include the video anymore right? since this is on the Explore page and we would want a different welcome type video here (in the future)

Reply:
This is what appears on the group about for a farm.

Reply:
Just for confirmation of that. This video will not appear as part of the Welcome Widget, just About:  ![Screen Shot 2022-05-05 at 9 57 46 PM](https://user-images.githubusercontent.com/6294/167070089-db33086a-1065-4b8b-947c-caebb5ca243e.png)